force-app/main/default/lwc/averageRating/averageRating.html

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
Instead of displaying "Unfortunately your browser does not support svg" would it be possible to instead just display a flat text value? Something like 3/5 or 2/5?

Reply:
Sure, the changes have been made